### PR TITLE
Fixing incident widgets

### PR DESCRIPTION
**Pull request by roysagi**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
In case you are a technological partner:
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

If you are not a technological partner (i.e. PANW employee, customer or individual developer), please delete this section.

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28717

## Description
Fixing the issue where the script names were misshaped - also fixing typo.

### CODE REVIEW

File: Packs/CommonWidgets/Scripts/GetLargestInputsAndOutputsInIncidents/GetLargestInputsAndOutputsInIncidents.yml [hunk 1]
```diff
@@ -13,10 +13,10 @@ args:
   secret: false
 comment: Returns inputs and outputs larger than 1 MB from all Cortex XSOAR investigations.
 commonfields:
-  id: GetLargestInputsAndOuputsInIncidents
+  id: GetLargestInputsAndOutputsInIncidents
```
Comment (ronykoz) [inline Packs/CommonWidgets/Scripts/GetLargestInputsAndOutputsInIncidents/GetLargestInputsAndOutputsInIncidents.yml:16]: don't change the id nor the name

**Comment by roysagi** [comment]
Loaded my update locally through the marketplace and it works:

**Comment by roysagi** [comment]
Loaded my update locally through the marketplace and it works:
image

File: Packs/CommonWidgets/Widgets/widget-LargestInputsAndOutputsInIncidentsWidget.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
- "id": "LargestInputsAndOuputsInIncidentsWidget",
+ "id": "LargestInputsAndOutputsInIncidentsWidget",
  "version": -1,
  "modified": "2020-08-18T13:31:21.939387+03:00",
- "name": "Largest Inputs And Ouputs In Incidents",
+ "name": "Largest Inputs And Outputs In Incidents",
```
Comment (ronykoz) [inline Packs/CommonWidgets/Widgets/widget-LargestInputsAndOutputsInIncidentsWidget.json:5]: let's keep just this change please

**Review by ronykoz — APPROVED** [review]

**Pull request merged**